scripts/abundance-dist.py

```diff
@@ -47,6 +47,9 @@ def get_parser():
                         + khmer.__version__)
     parser.add_argument('-f', '--force', default=False, action='store_true',
                         help='Overwrite output file if it exists')
+    parser.add_argument('--Header-names', dest='Header_names', default=False,
```

Comment:
I think this should be '--header-names'  - it's rare that long-form arguments have capital letters in them, for whatever reason.